### DISCUSSION TITLE
Is there a default limit on the max request header size?

### DISCUSSION THREAD

**Discussion by trallnag** [Kludex/uvicorn#1726] [Q&A] [answered]
N/T

**Comment by trallnag** [comment, marked as answer]
~~Currently there is no limit.~~

https://github.com/encode/uvicorn/issues/157

Edit: see comment by @Kludex for the actual answer

**Reply by Kludex** [reply]
There is for h11: `--h11-max-incomplete-event-size`.

Maybe the name is not very good...
